content/developers/node.md

```diff
@@ -52,11 +52,18 @@ A node can be started as a `daemon` by executing:
 mage daemon
 ```
 
-Or without mage:
+Or without (using a built binary) mage:
 ```bash
 build/myst/myst daemon
 ```
 
+This will start node in a `daemon` mode where no service is running, meaning that it will basically only operate in `consumer mode`. To start
+providing service you'd need to start a node with a running service (default is wireguard): 
+
+```bash
+build/myst/myst service
```

Comment:
maybe mention you should probably use sudo

Reply:
Don't have to. If you will need sudo, it will prompt you.

Reply:
yesterday I ran it without, I got prompted for it and there was some wireguad error which got solved by running with sudo. So not sure, it seemed okay without but connecting using cli was failing. @tomasmik

Reply:
Interesting, I'll take a look.